modules/ROOT/pages/deployment/general/roles-permissions.adoc

```diff
@@ -0,0 +1,128 @@
+
+=== Permission Granularity
+
+Compared to ownCloud Server where e.g. the administrator has more or less full access to everything, Infinite Scale refines roles with assigned permission types and particular permissions for each type on a much more detailed level. This is not only true for system roles but also for space roles.
+
+=== Security
+
+With role and permission separation (system/spaces/sharing), the overall security is greatly increased and security standards like *separating tasks and data access by design* are addressed.
+
+== System Roles and Permissions
+
+Currently there are four predefined system roles with fixed permissions. Via these roles, permissions are only granted to a particular user and not all users. Concatenating roles is currently not supported.
+
+Consider that these roles and permissions are provided by the system via APIs, which means you can call them via your own code or `curl`. 
+
+Read the image below the following way::
+* A Role gets one or more permission types assigned.
+* Each permission type grants individual rights relevant for that role.
+* Roles are assigned to users.
+
+// the empty macro is necessary to have more distance between text and image
+ 
+{empty} +
+
+// the table references following ocis code where roles and permissions a re defined: https://github.com/owncloud/ocis/blob/master/services/settings/pkg/store/defaults/defaults.go
+
+image::deployment/general/roles-and-permissions-matrix.svg[width=550]
+
+=== System Role Details
+
+In general with regards to spaces, the following applies:
+
+* Spaces can always be deleted by space managers - with the exception of the `Delete Home Space` permission, which is only assigned to the Admin since this should only happen when a user account gets deleted.
+* Managing a space means creating or deleting a space, setting quotas, delegating the right to manage space members and space member roles, but not having access to the content of the space except when being a member of the space.
+* When creating a space, the creator is automatically in the delegatable Manager role for this space. For details see xref:space-roles-and-permissions[Space Roles and Permissions].
+
+Admin::
+The `Admin` role provides full control of all permission types with one exception, which is `Self Management`. This permission type is excluded to avoid that the last remaining admin accidentally reduces their own role to a lower one making the system unmanageable. As long as there is more than one admin, roles can be set via the `Role Management` permission.
+
+Space Admin::
+The `Space Admin` role allows you to create spaces, set quotas and delegating the right to manage space members and space member roles.
+
+User::
+If you have the `User` role assigned, you can manage your own account and change the language. To manage space group members and space member roles, an Admin or Space Admin must create one and grant the Manager permission to the user.
```

Comment:
Do space groups exist? What does the (Space) Admin need to create?

Reply:
good point, need to check this